source/adios2/toolkit/remote/remote_common.h

```diff
@@ -105,6 +105,7 @@ typedef struct _CloseFileResponseMsg
 {
     int CloseResponseCondition;
     int Status;
+    size_t unused; // small messages call stack addressing issues?
```

Comment:
I would add in the comment the reason why you are adding this

Reply:
done